src/plugins/exec/sim-host-handlers.js

```diff
@@ -87,3 +96,19 @@ function handleSavedSim(success, fail, service, action) {
     }
     return false;
 }
+
+function sendExecUnhandledTelemetry(service, action, hasPersisted, isSuccess, hasResult, saveResult) {
+    var props = {
+        'service': service,
+        'action': action,
+        'has-persisted-result': hasPersisted,
+        'is-success': isSuccess,
+        'has-result-value': hasResult
+    };
+
+    if (typeof saveResult !== 'undefined') {
+        props['save-result'] = saveResult;
```

Comment:
What is the expected type of `saveResult`? From what I saw it seems to be a boolean, so maybe you can do the following:
```
props['save-result'] = !!saveResult; // defaults to false
```
Why this property is not always available in the `props` object? Is there any specific reason to not to do it?